### DISCUSSION TITLE
Is this project a fork of VLC

### DISCUSSION THREAD

**Discussion by Steperman5692** [huynhsontung/Screenbox#460] [Q&A] [answered]
So I was wondering if this project was a fork of VLC cause they in the description I see reference about VLC 🤔?

**Comment by huynhsontung** [comment, marked as answer]
Screenbox is not a fork of VLC, but it does use LibVLC as the media decoder/renderer. The app fetches the media file and feeds it to the VLC library to be displayed on the screen. I can totally swap the media decoder/renderer for another one like ffmpeg and Screenbox would still work.
